static/js/compose_actions.js

```diff
@@ -55,20 +56,49 @@ exports.set_focus = function (msg_type, opts) {
     }
 };
 
+function compose_not_subscribed_warning() {
+    const stream_name = compose_state.stream_name();
+    const sub = stream_data.get_sub(stream_name);
+    if (sub) {
+        const warning_text = render_compose_not_subscribed({
+            should_display_sub_button: sub.should_display_subscription_button,
+        });
+        $("#compose-send-status")
+            .removeClass(common.status_classes)
+            .addClass("home-error-bar")
+            .attr("stay", "true")
+            .stop(true)
+            .fadeTo(0, 1);
+        $("#compose-error-msg")[0].innerHTML = warning_text;
+        $("#compose-send-button").prop("disabled", false);
+        $("#sending-indicator").hide();
```

Comment:
These two lines are not required, the warning is independent of sending messages.